.github/workflows/build_and_test_midstream_image.yml

```diff
@@ -28,31 +27,27 @@ jobs:
         run: |
           podman --version
           cd Dockerfiles/midstream/
-          podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
-
-      - name: "Run example bundle through midstream_image -> POSITVE test"
+          podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0
+      - name: "Run example operator-metadata through midstream ci image  --> positive test"
+        continue-on-error: false
         run: |
-          cd Dockerfiles/midstream/
-          mkdir output_logs
-          podman run -it -v ./example-bundle:/project/operator-bundle -v ./output_logs:/project/output midstream_image:latest
-          # Test for files inside output_logs
-          test -f output_logs/validation-output.txt
-          test -f output_logs/validation-rc.txt
-          test -f output_logs/validation-version.txt
-          # clean output logs for next step
-          rm -rf output_logs/*
-
-      - name: "Run example bundle through midstream_image --> negative test"
+          mkdir ./output_logs
+          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output midstream_image:latest
+
+      - name: "Run example operator-metadata through midstream ci image  --> negative test"
         continue-on-error: true
         run: |
-          # cd into Dockerfiles since it resets everystep
-          cd Dockerfiles/midstream/
-          mkdir empty-example-bundle
-          podman run -it -v ./empty-example-bundle:/project/operator-bundle -v ./output_logs:/project/output midstream_image:latest
+          rm -rf ./output_logs
+          mkdir ./output_logs
+          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output -e IMAGE_TO_TEST=quay.io/somahesh/missing-alm-examples:v0.1.0 midstream_image:latest
```

Comment:
Should we move this test image `quay.io/somahesh/missing-alm-examples:v0.1.0` under quay.io/cvpops or its ok to have it as it is ?
This comment is just a question and has no intention to hold this MR ?

Reply:
That's a good point, I'll move the image to the cvpops/test-operator repository

Reply:
@sonam1412 moved the image, please take a look